### PR TITLE
A BRB for going to draw

### PR DESCRIPTION
**Pull request by KritRom**
This BRB allows the player to go and draw something, and if the affection is high enough she will also ask about what you drew. 
These include:

- Fanart
- Environments
- A real person
- An OC
- Something personal. (Player wants to keep it to themself)

As an artist I needed this, so I'm hoping this helps other artistic MAS players too.

Edit: In the latest version, the above menu has been removed to shorten the BRB. It also has different reactions for `broken` and `distressed`.

### CODE REVIEW

File: Monika After Story/game/script-brbs.rpy
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:1091]: Indentation error
```suggestion
    $ mas_idle_mailbox.send_idle_cb("drawing_brb_callback")
```

File: Monika After Story/game/script-brbs.rpy
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:1086]: This looks like normal+ dialogue rather than upset+

Additionally this conditional can be simplified to be more readable at a glance.
```suggestion
    if mas_isMoniNormal(higher=True):
```

File: Monika After Story/game/script-brbs.rpy
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:1090]: Ideally this should have some flows for upset and/or distressed.

`...` is reserved for `Broken`.

File: Monika After Story/game/script-brbs.rpy
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:1104]: Incorrect variable name
```suggestion
                if not persistent._mas_pm_drew_fanart:
```

File: Monika After Story/game/script-brbs.rpy
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:1095]: Same regarding clarity + aff levels.
```suggestion
    if mas_isMoniNormal(higher=True):
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Some code issues at a glance

I will say the dialogue seems a little long winded for a brb, if you want to add that dialogue I'd suggest having it in its own topic, which activates if you've used this brb (similarly to how the naps dialogue has a line that changes if you've used the nap brb)

**Comment by Kritzb278** [comment]
Ah okay I'll fix the errors you pointed out then. Thank you for your time :)

**Comment by Kritzb278** [comment]
I took the things you said into account and edited the dialogue accordingly. Was there anything more?

**Comment by Booplicate** [comment]
I'll add don't merge until the changes to the brb system are merged.

**Comment by Kritzb278** [comment]
Okay
Can you please tell me what that means (merging the changes to the brb system)?
Sorry I am very new to this

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1081]:
```suggestion
            unlocked=True
        ),
```

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1077]:
```suggestion
            prompt="I'm going to go draw",
```

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1089]: Perhaps you mean this? You need to actually call functions.
```suggestion
    elif mas_isMoniBroken():
```

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1094]: Indentation mismatch
```suggestion
    return "idle"
```

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1103]: Must call functions
```suggestion
    elif mas_isMoniBroken():
```

File: Monika After Story/game/script-brbs.rpy
Comment (Booplicate) [inline Monika After Story/game/script-brbs.rpy:1107]: Indentation mismatch
```suggestion
    return
```

**Review by Booplicate — CHANGES_REQUESTED** [review]
You shouldn't worry about that. I marked more things for you to fix.

**Comment by Kritzb278** [comment]
Okay I think I made all the changes you requested. I hope it's good now

**Comment by Booplicate** [comment]
You can use the `Add suggestion to batch` button and then commit all the changes. That way you won't miss the suggestions.

**Comment by Kritzb278** [comment]
Ah okay thank you
Sorry I have no idea how all of this works

**Comment by Kritzb278** [comment]
Um I hope this doesn't sound rude but can someone remove the Awaiting Changes tag? I think I have changed all that I was asked to so yeah

**Pull request closed without merging**
